### DISCUSSION TITLE
Can a user I blocked create a comment on a discussion thread that I created on a public repository?

### DISCUSSION THREAD

**Discussion by partingscientist** [community/community#116651] [Discussions] [answered]
### Select Topic Area

Question

### Body

According Github Docs:

> After you've blocked a user, they cannot:
> ...
> Comment on or edit issues or pull requests that you've created

Does this include discussion threads that I created on public repositories?

**Comment by 0YJ** [comment, marked as answer]
No, he can'tcomment on a discussion thread that you created in a public repository.

**Comment by Xelliese** [comment]
Yep, once you block someone, they can't comment or edit anything on your threads, even in public repositories. Blocking pretty much stops them from interacting with your stuff on the forum.

**Comment by Smail1223344** [comment]
<details><summary>Details</summary>
<p>

1. [ ] 

</p>
</details>

**Comment by Smail1223344** [comment]
1. @0YJ

**Comment by Smail1223344** [comment]
``
